dockers/docker-database/multi_database_config.json.j2

```diff
@@ -0,0 +1,192 @@
+{% set include_remote_db = (REMOTE_DB_IP is defined and REMOTE_DB_PORT is defined) %}
+{
+    "INSTANCES": {
+        "redis":{
+            "hostname" : "{{HOST_IP}}",
+            "port" : {{REDIS_PORT}},
+            "unix_socket_path" : "/var/run/redis{{DEV}}/redis.sock",
+            "persistence_for_warm_boot" : "yes"
+            {% if DATABASE_TYPE is defined and DATABASE_TYPE != "" %}
+            ,"database_type": "{{DATABASE_TYPE}}"
```

Comment:
I didn't find code in sonic-swss-common to parse and handle 'database_type', is there a PR?

Reply:
https://github.com/sonic-net/sonic-buildimage/pull/5283

Reply:
I search that PR, not find any code read "database_type"
This attribute is an optional attribute, and this file need be parsed by SonicDBConfig::parseDatabaseConfig
https://github.com/sonic-net/sonic-swss-common/blob/master/common/dbconnector.cpp

Otherwise, sonic-swss-common lib can't handle multi-DB, so please make sure sonic-swss-common also change to handle it.

Reply:
In  /sonic-buildimage/dockers/docker-database/docker-database-init.sh line 53,

Reply:
@liuh-80 

This field was added via 
https://github.com/sonic-net/sonic-buildimage/commit/9f08f88a0d1e45cdab3b1e5037a24908057e6129
by @Pterosaur

It is current SONiC behavior
https://github.com/sonic-net/sonic-buildimage/blob/master/dockers/docker-database/database_config.json.j2#L10

Can you help to check with him to see what the need for this field?

Reply:
#Closed, I will offline sync with Ze